procedures/auxtel/reduction_procedures/Parabolic_Focus_Analysis.ipynb

```diff
@@ -0,0 +1,585 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    focusAnalyzer.setSpectrumBoxOffsets([500, 750, 1000, 1250])
</pre><p>I think it's worth adding a sentence just explaining what this box does. Basically setting where the gaussians are fit. So the user should must manually make sure they overlap the spectrum etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Added some more explanation for the focusAnalyzers.